roadmap/implementors-guide/src/types/backing.md

```diff
@@ -38,28 +67,39 @@ enum Statement {
   /// A statement about the invalidity of a candidate.
   Invalid(Hash),
 }
+
+/// A statement about the validity of a parachain candidate.
+///
+/// This variant should only be used in the production of `SignedStatement`s. The only difference between
+/// this enum and `Statement` is that the `Seconded` variant contains a `Hash` instead of a `CandidateReceipt`.
+/// The rationale behind the difference is that a `CandidateReceipt` contains `HeadData`, which does not have
+/// bounded size. By using this enum instead, we ensure that the production and validation of signatures is fast
+/// while retaining their necessary cryptographic properties.
```

Comment:
```suggestion
/// The rationale behind the difference is that the signature should always be on the hash instead of the
/// full data, as this lowers the requirement for checking while retaining necessary cryptographic properties
```

This documentation is jumping to conclusions, better to keep it simple (I'm not really worried about head data sizes).